bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/PCMConverter.java

```diff
@@ -303,9 +359,13 @@ private Node createCorrespondingDFDNode(AbstractPCMVertex<? extends Entity> pcmV
         Behaviour behaviour = datadictionaryFactory.eINSTANCE.createBehaviour();
 
         node.setEntityName(computeCompleteName(pcmVertex));
-        node.setId(pcmVertex.getReferencedElement()
-                .getId());
-
+        
+        var id = pcmVertex.getReferencedElement()
+                .getId();
+        while (takenIds.contains(id)) id += "_1";
```

Comment:
If i interpret this correctly, this does id_1_1_1_1_1_1 rather the id_6 what would be expected

Reply:
Fixed in 9c95822ed25c